samples/dnn/text_detection.cpp

```diff
@@ -5,24 +5,31 @@
 using namespace cv;
 using namespace cv::dnn;
 
+
 const char* keys =
     "{ help  h     | | Print help message. }"
     "{ input i     | | Path to input image or video file. Skip this argument to capture frames from a camera.}"
-    "{ model m     | | Path to a binary .pb file contains trained network.}"
+    "{ model m     | | Path to a binary .pb file contains trained detector network.}"
+    "{ ocr         | | Path to a binary .pb or .onnx file contains trained recognition network.}"
```

Comment:
Please add model conversion code to the sample (in comments at the top of file)

Reply:
Please add links to text detection and optional text recognition models into this file (as comment at the top).